docs/pods/networking.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Global Networking"
+description: "Learn how to securely communicate between pods within your RunPod account using a private network."
+id: networking
+sidebar_position: 6
+---
+
+This private networking feature enables secure communication between all Pods within your RunPod account. It creates a private, virtual network that connects your Pods, allowing them to communicate with each other as if they were on the same local network, regardless of their physical location.
+
+## How to use it?
+
+1. **Enable Global Networking**:
+
+   - When creating a new Pod, check the "**Global Networking**" option.
+
+![How to enable Global Networking for your Pod](/img/docs/global-networking-enable.png)
+
+2. **Access the Private Network**:
+   - Each Pod with Global Networking enabled will be assigned a private IP address.
+   - The Private IP Address is referenced via the DNS record for the pod using the pattern: $podid.runpod.internal
+   - The DNS record will displayed on the pod details card after the pod is created.
+3. **Run Services**:
+   - Start your services on the Pods as usual. They will be accessible to other Pods on the private network without needing to explicitly expose ports.

```

Comment:
Maybe add one more step: 
to communicate with your other pods, open a terminal and run `ping podid.runpod.internal`.